src/widgets/panel/panel.tsx

```diff
@@ -77,14 +79,16 @@ export type PanelTemplates<TWidget extends WidgetFactory<any, any, any>> = {
 };
 
 export type PanelRenderOptions<
-  TWidget extends WidgetFactory<any, any, any>
+  TWidget extends WidgetFactory<unknown, unknown, unknown>
```

Comment:
we should have an intermediate type in this file `UnknownWidgetFactory` instead of repeating this so many times imo

Reply:
I can do that, but can you help me with this? https://github.com/algolia/instantsearch.js/pull/4527#discussion_r515983857